clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/security/KubernetesV2Credentials.java

```diff
@@ -115,6 +115,14 @@ public String getDefaultNamespace() {
     return cachedDefaultNamespace;
   }
 
+  public List<String> getNamespaces() {
```

Comment:
I would just annotated the `namespaces` and `omitNamespaces` fields with `@Getter` like the properties above around line 77